pkg/datapath/linux/node.go

```diff
@@ -811,6 +805,16 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 	return nil
 }
 
+// getExternalFacingDeviceName returns the external / cluster-facing device
```

Comment:
Hmm, not sure about naming of this method. In the case of `DirectRoutingDevice` it might be not external facing device, as it's used for internal communication between k8s nodes.

Reply:
Any ideas for a name?

Reply:
Maybe just `getNodeDeviceName`?